packages/react-native/Libraries/Components/Pressable/Pressable.js

```diff
@@ -408,12 +408,17 @@ function Pressable(
       // $FlowFixMe[prop-missing]
       if (evt.eventType === 'select') {
         // $FlowFixMe[incompatible-exact]
+        onPressIn(evt);
```

Comment:
Sorry for the late review but I wonder if we should solve this problem on the native layer by sending the expected events for [Pressability](https://github.com/react-native-tvos/react-native-tvos/blob/tvos-v0.76.0/packages/react-native/Libraries/Pressability/Pressability.js) to do its job. Here we're introducing our own way of doing the same thing which is a bit confusing and redundant if you ask me

Reply:
I definitely see your point. NativeWind is using its own package called `react-native-css-interop`, which is looking for `onPressIn/Out` events. Ideally, we should be trying to support all these events, and maybe even be able to bring in the `Touchable` components too if we can. I'll take another look at this to see if we can move these changes to the `Pressability` module.

Reply:
One complication that arises in trying to handle this purely from the native layer, is that on Apple TV there are no key up/key down events like in Android TV, that can be directly mapped to `onPointerUp/Down`. What you have instead are gesture recognizers that fire handlers when a select or longSelect is detected. For longSelect, you do get two events, one at the start and one at the end, so that is no problem. For just a select, you only get one event, so we would have to generate the pointer up/down events synthetically anyway....

Reply:
FYI @yusufyildirim I went ahead and reverted this change in tvos-v0.76.0 branch and in the 0.76.0-0rc3 release that will be published tomorrow.